pkg/evaluation/jobs.go

```diff
@@ -40,6 +40,7 @@ const (
 	defaultPVCMountPath = "/data/evaluations"
 	defaultTestRagFile  = "ragas.csv"
 	defaultMCImage      = "kubeagi/minio-mc:RELEASE.2023-01-28T20-29-38Z"
+	defaultEvalImage    = "kubeagi/arcadia-eval:v0.2.0"
```

Comment:
how v0.2.0 got built?

Reply:
Identical. 
https://hub.docker.com/layers/kubeagi/arcadia-eval/v0.1.0/images/sha256-dbde5aa5c25e6464f3226031ec220631cd1385c1073100ce9f658c75087c7cf1?context=explore

https://hub.docker.com/layers/kubeagi/arcadia-eval/v0.2.0/images/sha256-66c8506bf663d3b8be3c6d0e24720a69d733fc9907bc5020a2d04eea57305311?context=explore

We re-tagged a v0.2 image to v0.1 and pushed to hub 2 weeks ago due to this value setting issue. Rename this ( or considerably make it some env param) is a better option.

Reply:
we can keep v0.1.0 image tag unless image really get updated